### DISCUSSION TITLE
Question: Excluding '@eaDir' Directory in Synology from Stash Using Regex

### DISCUSSION THREAD

**Discussion by onlyhooops** [stashapp/stash#5147] [Q&A] [answered]
How to exclude the '@eaDir' directory in Synology systems from stash? I've configured a regular expression but it doesn't seem to be working. I hope someone can tell me the correct way to do it. Thank you very much.

**Comment by DogmaDragon** [comment, marked as answer]
https://github.com/stashapp/stash/issues/3171 has the regex mentioned, that should work.

**Reply by onlyhooops** [reply]
thanks!
